soapbox/general/smonn-efficiency.md

```diff
@@ -0,0 +1,59 @@
+# Efficiency as a programmer
+
+📝 By [Simon][smonn]
+
+[smonn]: https://github.com/smonn
+
+---
+
+Someone once said that a great programmer is a lazy one. The key being that they try to get as much done with as little effort as possible as opposed to not getting anything done. In other words, they are efficient. So what can you do to improve your efficiency?
+
+## TL;DR
+
+- Learn and use keyboard shortcuts
+- Write less/more code
+- Ask questions early and often
+- Sometimes you just gotta RTFM
+- Always be learning
+
+## Learn and use keyboard shortcuts
+
+This applies to any editor (or IDE). As soon as you find yourself doing something repetitive that _feels_ slow, try to find out if there's a keyboard shortcut for it. If not, can you add one?
+
```

Comment:
I feel like it's worth a mention to invest time in learning a keyboard based editor such as Vi or Emacs here.

Reply:
Probably for over-achievers, but it is useful to know some basic terminal-based editor like Vi[m] (or even Nano). Never know when you end up on a server via SSH and need to edit that one config file... So will add a mention of this!